nagios/tox.ini

```diff
@@ -2,16 +2,20 @@
 minversion = 2.0
 basepython = py37
 envlist =
-    py{27,37}-{unit,nagios}
+    py{27,37}-latest
 
 [testenv]
+description =
+    py27,py37: e2e ready
 dd_check_style = true
 usedevelop = true
 platform = linux|darwin|win32
 deps =
     -e../datadog_checks_base[deps]
     -rrequirements-dev.txt
+passenv =
+    DOCKER*
+    COMPOSE*
 commands =
     pip install -r requirements.in
-    unit: pytest -m"not integration" -v
-    nagios: pytest -m"integration" -v
+    pytest -v
```

Comment:
(nit) missing ending newline. (maybe we should add this to the linter/formatter).